bin/postman-test/collections/system-management.postman_collection.json

```diff
@@ -0,0 +1,1006 @@
+{
+	"info": {
+		"_postman_id": "a7a08cc2-6b9c-4dd4-a3ae-d1d66feef471",
+		"name": "sma",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "config",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "a33f91fa-2e51-46f7-8c8d-30e4487ca744",
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"pm.test(\"Response time is less than 600ms\", function () {",
+							"    pm.expect(pm.response.responseTime).to.be.below(600);",
+							"});",
+							"",
+							"pm.test(\"Content-Type is application/json\", function(){",
+							"    var contentType = pm.response.headers.get('Content-Type');",
+							"    pm.expect(contentType).to.include('application/json');",
+							"});",
+							"",
+							"tests[\"Status code is 200\"] = responseCode.code === 200;",
+							"  if(responseCode.code === 200){",
+							"      tests[\"Contains Configuration\"] = responseBody.has(\"Configuration\");",
+							"      tests[\"Contains edgex-core-command\"] = responseBody.has(\"edgex-core-command\");",
```

Comment:
@cloudxxx8 
- Please confirm whether the assertions I've added (such as these ones, to validate the response content which contains the required information) align with the blackbox testing approach. 
- If so, I will add further assertions: Note that I have added a set of assertions (these ones) as an example for you to examine 👍